WolvenKit.App/ViewModels/Tools/ChunkViewModelExtended/ChunkViewModel.SearchAndReplace.cs

```diff
@@ -32,17 +32,19 @@ public static void SearchAndReplace_ResetCaches()
     /// <param name="replace"></param>
     /// <param name="isWholeWord"></param>
     /// <param name="isRegex"></param>
+    /// <param name="ignoreCache"></param>
     /// <returns></returns>
-    private int SearchAndReplaceInternal(string search, string replace, bool isWholeWord, bool isRegex)
+    private int SearchAndReplaceInternal(string search, string replace, bool isWholeWord, bool isRegex,
+        bool ignoreCache = false)
     {
         NumReplacedEntries = 0;
         var isExpanded = IsExpanded;
 
         // SearchAndReplaceInProperties can influence the expansion state, even if nothing was changed
-        var hasReplacement = SearchAndReplaceInProperties(search, replace, isWholeWord, isRegex);
+        var hasReplacement = SearchAndReplaceInProperties(search, replace, isWholeWord, isRegex, ignoreCache);
         IsExpanded = isExpanded;
```

Comment:
Why set it back here and also before? It's not clear where IsExpanded is set

Reply:
isExpanded will store the node's expansion state to restore it after potential replacements (this will always expand the node).